d2bs/kolbot/libs/common/Pather.js

```diff
@@ -572,7 +572,7 @@ ModeLoop:
 		clearPath - kill monsters while moving
 	*/
 	moveToExit: function (targetArea, use, clearPath) {
-		var i, j, area, exits, targetRoom, dest, currExit,
+		var i, j, area, exits, targetRoom, dest, currExit, unit, lever,
```

Comment:
"unit" is a generic variable name, use something more specific. There are various examples in existing kolbot code.

Reply:
Other functions seem to use it in a similar manner, changing it to something more specific may not be useful.